focus-ios/focus-ios-tests/XCUITest/SettingTest.swift

```diff
@@ -102,7 +102,11 @@ class SettingTest: BaseTestCase {
         let reviewCell = app.cells["settingsViewController.rateFocus"]
         let safariApp = XCUIApplication(privateWithPath: nil, bundleID: "com.apple.mobilesafari")!
 
-        app.tables.firstMatch.swipeUp()
+        var swipes = 3
+        while !reviewCell.isHittable && swipes > 0 {
```

Comment:
@isabelrios We had a conversation on `isHittable`, `isEnabled` and `exists`. Generally, `exists` works quite well, but existence does not guarantee `isHittable`. `isEnabled` may be better.

Reply:
@clarmso does isEnabled guarantee the element isHittable?

Reply:
I did a quick test and having the element enabled doesn't mean it is visible or hittable and we need to tap on it.